src/Interpreters/QueryLog.h

```diff
@@ -58,6 +58,15 @@ struct QueryLogElement
     std::set<String> query_tables;
     std::set<String> query_columns;
 
+    std::set<String> factory_aggregate_functions;
```

Comment:
Let's rename columns to `used_...` instead of `factory_...` - it will be more understandable.